tests/test_renderer_style.py

```diff
@@ -1,45 +1,18 @@
-from markdown_it import MarkdownIt
+from pathlib import Path
+
+from markdown_it.utils import read_fixture_file
 import pytest
 
-from mdformat.renderer import MDRenderer
+from mdformat import text as render
```

Comment:
What's the reason for this change? I would perhaps prefer to leave this as a test for the `renderer` package. There is another one for the public API functions.

Reply:
I've changed it pack to using the renderer, but note you have to add:

```python
md_new.options["store_labels"] = True
```

for the references to be recovered (see https://github.com/executablebooks/markdown-it-py/commit/d9b6de9c8ff3bb0dc2b78d6eda8e09b3b88fa7b4)

Reply:
Ah alright. What happens if `store_labels = False` now? The ref links just get formatted as inline (which would seem fine). If it's something more unexpected, we might want to just fail in `MDRenderer.__init__` with a ValueError if store_labels isnt enabled.

Reply:
yep they just get formatted as inline 👍